### ISSUE TITLE
Login with SSO

### ISSUE THREAD

**Issue by djaiss** [officelifehq/officelife#363]
We should be able to login using brand login
- LDAP
- SAML
- Active Directory

**Comment by djaiss** [comment]
@asbiin I agree. It’s extremely important for a project like OfficeLife to connect to AD or similar and manage users that way.

That being said, I have no clue on how to do it.

This project, https://github.com/kanboard/kanboard/tree/master/app/Core/Ldap, has written everything on their own.

**Comment by cweagans** [comment]
IMO, if you _only_ do SAML to start with, that covers 80% of SSO use-cases out of the gate. Google Apps can act as a SAML identity provider, ADFS is a SAML identity provider that is included with Active Directory, and there is at least one open source SAML identity provider that can use LDAP as a user store (simpleSAMLphp).

I wrote a SAML integration for Drupal (https://drupal.org/project/samlauth). Happy to help with a knowledge download on this one if you'd like. SAML is very straightforward to implement once you get past all of the jargon.

**Comment by evrardjp** [comment]
SAML would be a great idea, LDAP would be a good second choice.

**Comment by sama8** [comment]
If relevant -> [SAML Jackson](https://github.com/boxyhq/jackson) 🚀

**Comment by earthpyy** [comment]
Would love to see this implemented!
